scripts/verify_image_sign_test.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Can we move all the test script in the tests folder?

Reply:
Where do you suggest to move these test to?

Reply:
The script scripts/verify_image_sign_test.sh is used by tests/sign_and_verify_test.py, can we move the script verify_image_sign_test.sh to the tests/ folder?